### DISCUSSION TITLE
Dasel V3 early adopters

### DISCUSSION THREAD

**Discussion by TomWright** [TomWright/dasel#439] [General]
I've been working in the background on a new version of dasel.

I've made what I would consider a lot of major improvements, and am almost at the point where I'd like to get some early adopters to run the latest version.

The idea for these early adopters would be:
1. Iron out any issues that I've missed.
2. Verify the syntax works and is as intuitive as I hope.
3. Gather suggestions/improvements that I can implement before it's released.

I'm currently working on the documentation over at https://daseldocs.tomwright.me/v3

Is anyone interested in helping me make this next major version as good as possible?

**Comment by yonas** [comment]
When do you plan on releasing v3?

**Reply by TomWright** [reply]
The goal is asap. Dasel development has taken a bit of a back seat recently due to real life commitments but it is most of the way there

**Comment by TomWright** [comment]
Was anyone able to install and test out V3 as it stands? Any thoughts on the syntax, any issues?

**Comment by TomWright** [comment]
V3 is now merged to master

**Comment by TomWright** [comment]
V3 has now been released.

**Reply by yonas** [reply]
> V3 has now been released.

Congratulations! 🎊
